proxy/src/main/java/com/velocitypowered/proxy/command/VelocityCommandManager.java

```diff
@@ -155,19 +154,18 @@ public boolean executeImmediately(final CommandSource source, final String cmdLi
     Preconditions.checkNotNull(source, "source");
     Preconditions.checkNotNull(cmdLine, "cmdLine");
 
-    ParseResults<CommandSource> results = parse(cmdLine, source, true);
+    String normalizedInput = BrigadierUtils.normalizeInput(cmdLine, true);
+    String[] args = normalizedInput.split(" ");
```

Comment:
Instead of splitting, we can do the complement of https://github.com/VelocityPowered/Velocity/blob/e8abdba4a6f54d1fa0a4d793d994adebdf48957c/proxy/src/main/java/com/velocitypowered/proxy/util/BrigadierUtils.java#L85 to avoid the array allocation + entire string lookup.